tensorflow/python/framework/importer_test.py

```diff
@@ -219,6 +219,72 @@ def testBasic(self):
       # Check that the op_def is still available.
       self.assertNotEqual(None, a.op_def)
 
+      # Check that the Ops were placed in the graph
```

Comment:
Added some tests to `testBasic` to make sure the added Ops were in the default graph.